apps/condo/schema/Billing.test.js

```diff
@@ -22,6 +25,62 @@ async function createBillingIntegration (client, extraAttrs = {}) {
     return [obj, attrs]
 }
 
+async function createBillingIntegrationAccessRight (client, user, integration, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
```

Comment:
Later we discussed about this raise exception pattern and we conclude that we will use the Facebook library for this purpose.